src/users/UserList.js

```diff
@@ -56,7 +56,6 @@ export default class UserList extends PureComponent<Props> {
             fullName={item.full_name}
             avatarUrl={item.avatar_url}
             email={item.email}
-            presence={presences[item.email]}
```

Comment:
This hunk of the diff made me a bit suspicious -- why is it unlike all the others in that we delete this one line without any others?

Looking, there's at least this line that probably already did nothing:
```
        exraData={presences}
```

But then we do appear to use `presences` for sorting and grouping the users, so that may be it.

Reply:
This would have had effect if we did not already inject the `presence` inside `Avatar` previously and `PresenceStatusIndicator` now...

Also good that we did not need it even before that, as this is called `exra` as in '`extra` without the `t`' :)
Removing this.